srcpkgs/nix/template

```diff
@@ -1,16 +1,15 @@
 # Template file for 'nix'
 pkgname=nix
-version=1.11.16
-revision=4
+version=2.0.4
+revision=1
 build_style=gnu-configure
 # Use /nix/var as suggested by the official Manual.
 configure_args="--localstatedir=/nix/var"
-depends="curl perl-WWW-Curl perl-DBD-SQLite"
-hostmakedepends="automake libtool pkg-config flex ${depends}"
+depends="curl"
+hostmakedepends="autoconf automake libtool pkg-config flex ${depends}"
 # Extra run-time dependencies.
```

Comment:
this comment should be above depends= shouldn't it ?

Reply:
not sure, since it isn't "extra"

maybe remove since perl is gone?